src/Pagehead.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import {COMMON, get} from './constants'
+import theme from './theme'
+import sx from './sx'
+
+const StyledPagehead = styled.div`
+  position: relative;
+  padding-top: ${get('space.4')};
+  padding-bottom: ${get('space.4')};
+  margin-bottom: ${get('space.4')};
+  border-bottom: 1px solid ${get('colors.border.gray')};
+  ${COMMON};
+  ${sx};
+`
+
+const Pagehead = ({...props}) => {
```

Comment:
Since we don't need to do anything special here, we can remove lines 18-20 and rename the component in line 8 `Pagehead`! 🎉

Reply:
Just fixed it, thank you for the suggestion!